website/app/components/doc/scroll-to-top/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class DocScrollToTopComponent extends Component {
+  @service fastboot;
+  @tracked isButtonVisible = false;
+
+  constructor() {
+    super(...arguments);
+    this.addScrollListener();
+  }
+
+  willDestroy() {
+    super.willDestroy(...arguments);
+    this.removeScrollListener();
+  }
+
+  @action
+  scrollToTop() {
+    if (!this.fastboot.isFastBoot) {
+      window.scrollTo({
+        top: 0,
+        behavior: 'smooth',
+      });
+    }
```

Comment:
you don't need to check for fastboot here, it's an action invoked by the user
```suggestion
    window.scrollTo({ top: 0, behavior: 'smooth' });
```